cnv/cnv_users_guide/cnv-ansible-modules.adoc

```diff
@@ -0,0 +1,15 @@
+[id="ansible-modules"]
+= Automating management tasks using Ansible
+include::modules/cnv-document-attributes.adoc[]
+:context: cnv-ansible-modules
```

Comment:
```suggestion
```

Reply:
We use this in assemblies to build the direct link to the module. The [id="cnv-ansible-modules_{context}"] gets its context from the :context: line of the assembly that the module is in.

Currently, there's no assembly that includes this module, but I'll create one as a follow-up PR.